extensions/plugins/system/redcore/redcore.php

```diff
@@ -119,6 +119,8 @@ public function onAfterInitialise()
 							'absoluteHrefs'     => $input->get->getBool('absoluteHrefs', true),
 						);
 
+						$this->cleanCache();
```

Comment:
I'm not sure what we would gain by adding this into the API itself, could you explain